app/lib/pgx/pages/pgx.dart

```diff
@@ -89,4 +95,16 @@ class PgxPage extends StatelessWidget {
       ),
     );
   }
+
+  void _scrollToSelectedContent({required GlobalKey key}) {
+    final keyContext = key.currentContext;
+    if (keyContext != null) {
+      Future.delayed(Duration(milliseconds: 200)).then((value) {
```

Comment:
```suggestion
      Future.delayed(Duration(milliseconds: 200)).then((value) {
```
Code is good, works fine as well

But there are two discussions i feel like we could have about this
1) Could we automatically scroll to the extended question as it is extended (scratch the first 200 ms delay?)
2) When one of the cards is maximized, should we minimize the one that is currently maximized?